src/components/Preview.jsx

```diff
@@ -26,7 +26,12 @@ class Preview extends React.Component {
 
     return generatePreview(
       project,
-      {targetBaseTop: true, propagateErrorsToParent: true, breakLoops: true}
+      {
+        targetBaseTop: true,
+        propagateErrorsToParent: true,
+        breakLoops: true,
+        isPopout,
```

Comment:
I like keeping the `generatePreview` method unaware of interface-level concepts like a popout—would be inclined to name this `nonBlockingAlerts` or somesuch